wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -60,6 +67,25 @@
 S3_MAX_MULTI_UPLOAD_SIZE = 5 * 1024**4
 
 
+# Minimum size to switch to multipart download, same as upload, 2GB.
+_MULTIPART_DOWNLOAD_SIZE = S3_MIN_MULTI_UPLOAD_SIZE
+# Multipart download part size is same as multpart upload size, which is hard coded to 100MB.
+# https://github.com/wandb/wandb/blob/7b2a13cb8efcd553317167b823c8e52d8c3f7c4e/core/pkg/artifacts/saver.go#L496
+# https://docs.aws.amazon.com/AmazonS3/latest/userguide/optimizing-performance-guidelines.html#optimizing-performance-guidelines-get-range
+_DOWNLOAD_PART_SIZE_BYTES = 100 * 1024 * 1024
+# Chunk size for reading http response and writing to disk. 1MB.
+_HTTP_RES_CHUNK_SIZE_BYTES = 1 * 1024 * 1024
+# Signal end of _ChunkQueue, consumer (file writer) should stop after getting this item.
+_CHUNK_QUEUE_SENTIENL = object()
```

Comment:
```suggestion
_CHUNK_QUEUE_SENTINEL = object()
```
I'm assuming you were going for Sentinel?